### PR TITLE
Fixes for list type field.

### PR DESCRIPTION
**Pull request by codeart1st**
Changed my profile from n3tfr34k to codeart1st ;) More fixes coming.

### CODE REVIEW

**Comment by n3tfr34k** [comment]
Windows git integration ist bad. The full commit message is:

#719 The query select both input fields (helper and original) and try to focus both. But we want focus only the first one.

File: src/w2utils.js [hunk 1]
```diff
@@ -1332,7 +1332,7 @@ w2utils.event = {
             '<div id="w2ui-overlay'+ name +'" style="display: none; left: 0px; top: 0px; '+ options.overlayStyle +'"'+
             '        class="w2ui-reset w2ui-overlay '+ ($(this).parents('.w2ui-popup, .w2ui-overlay-popup').length > 0 ? 'w2ui-overlay-popup' : '') +'">'+
             '    <style></style>'+
-            '    <div style="'+ options.style +'" class="'+ options['class'] +'"></div>'+
+            '    <div style="min-width: 100%; '+ options.style +'" class="'+ options['class'] +'"></div>'+
```
Comment (vitmalina) [inline src/w2utils.js]: I do not undestand what remove "min-width". This div is inside overlay and should take entire width

Comment (codeart1st) [inline src/w2utils.js]: This happened in chrome in our setup with list field in a w2ui form. Perhaps it was a conflict with our CSS environment. I will check this again next week.

Comment (codeart1st) [inline src/w2utils.js]: @vitmalina know I remember me why I added this style property. The real bug is a wrong inline style width in the inner div. The simplest solution was the adding min-width: 100% to this inner div. Look at the pictures below.

Bug

DOM

Comment (vitmalina) [inline src/w2utils.js]: So, is "min-width" needed?

Comment (codeart1st) [inline src/w2utils.js]: @vitmalina perhaps we can fix the wrong child width calculation. But using min-width instead of it is not bad in my opinion.

Comment (vitmalina) [inline src/w2utils.js]: I was fixing some other bug in w2overlay and was testing it. It appears that all is working correctly. I will keep it as is for now. Thanks.

**Pull request merged**